agents/PCS-Large-LP-Deposit-Withdrawal/src/agent.spec.ts

```diff
@@ -0,0 +1,298 @@
+import { Finding, FindingSeverity, FindingType, HandleTransaction, TransactionEvent } from "forta-agent";
+import { BigNumber, utils } from "ethers";
+import { createAddress, TestTransactionEvent } from "forta-agent-tools/lib/tests";
+import { Interface } from "@ethersproject/abi";
+import { when } from "jest-when";
+import { provideHandleTransaction } from "./agent";
+import { EVENTS_ABI } from "./constants";
+import { getCreate2Address } from "@ethersproject/address";
+
+const MOCK_IFACE: Interface = new Interface(EVENTS_ABI);
+const MOCK_FACTORY: string = createAddress("0x1111");
+
+const MOCK_POOL_SUPPLY_THRESHOLD: BigNumber = BigNumber.from(10000);
+const MOCK_THRESHOLD_PERCENTAGE: BigNumber = BigNumber.from(10);
+
+// valid, token0, token1, amount0, amount1, totalSupply
+const CASES: [boolean, string, string, string, string, BigNumber][] = [
+  [
+    true,
+    createAddress("0xaa11"),
+    createAddress("0xbb22"),
+    "100000000000000000",
+    "200000000000000",
+    BigNumber.from(3534953498534),
+  ],
+  [
+    true,
+    createAddress("0xcc33"),
+    createAddress("0xdd44"),
+    "200000000000000000",
+    "300000000000000",
+    BigNumber.from(100000),
+  ],
+  [
+    true,
+    createAddress("0xcccc"),
+    createAddress("0xdddd"),
+    "300000000000000000",
+    "400000000000000",
+    BigNumber.from(13534953498534),
+  ],
+  [
+    true,
+    createAddress("0xeeee"),
+    createAddress("0xffff"),
+    "8700000000000000000",
+    "2500000000000000",
+    BigNumber.from(3534953498534),
+  ],
+  [
+    true,
+    createAddress("0xccdd99"),
+    createAddress("0xeeff00"),
+    "4400000000000000001",
+    "5500000500000700",
+    BigNumber.from(3534953498534),
+  ],
+];
+
+// valid, token0, token1, amount0, amount1, totalSupply
+const FAIL_CASES: [boolean, string, string, string, string, BigNumber][] = [
+  [
+    true,
+    createAddress("0xc459"),
+    createAddress("0x2222"),
+    "100", //low amount0
+    "200", //low amount1
+    BigNumber.from(200000),
+  ],
+  [
+    true,
+    createAddress("0xaaab"),
+    createAddress("0xbaaa"),
+    "200000000000000000",
+    "300000000000000",
+    BigNumber.from(7500), // low totalSupply
+  ],
+];
+
+const testBlocks: number[] = [341530, 341531, 341532];
```

Comment:
Use uppercase for all constants.
```suggestion
const TEST_BLOCKS: number[] = [341530, 341531, 341532];
```

Reply:
Done